src/modules/Favorites/reducers/index.js

```diff
@@ -0,0 +1,79 @@
+import {
+  REMOVE_FAVORITE_ALBUM,
+  SET_FAVORITE_ALBUM,
+  SET_FAVORITE_TRACK,
+  REMOVE_FAVORITE_TRACK,
+} from '../../../constants/actionsTypes';
+
+export const INITIAL_STATE = {
+  favorites: {},
+};
+
+export const favoriteAlbumsReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case SET_FAVORITE_ALBUM: {
+      const newFavorites = {
+        ...state.favorites,
+        [action.payload.albumId]: {
+          album: action.payload.album,
```

Comment:
A lot of action.payload mentions. Object destructuring needed (Please, check all file. Pay attention on empty lines before return)